tests/validation/tests/v3_api/common.py

```diff
@@ -1031,16 +1045,8 @@ def validate_dns_entry(pod, host, expected, port=TEST_IMAGE_PORT, retry_count=3)
     if HARDENED_CLUSTER:
         cmd = 'curl -vs {}:{} 2>&1'.format(host, port)
     else:
-        cmd = 'ping -c 1 -W 1 {0}'.format(host)
-    cmd_output = kubectl_pod_exec(pod, cmd)
-
-    if str(pod.name) not in str(cmd_output):
-        for i in range(0, retry_count):
-            cmd_output = kubectl_pod_exec(pod, cmd)
-            if str(pod.name) in str(cmd_output):
-                break
-            else:
-                time.sleep(5)
+        cmd = 'ping -c 2 -W 2 {0}'.format(host)
```

Comment:
If i understand correctly we're delegating the retry and validation in `retry_cmd` now so I would say we should be more clear in the method name at what's doing. Like the `retry_cmd` to `retry_cmd_validate_expected` (this is just a naming idea, not a request).

Reply:
Right, that makes more sense. I changed the `retry_cmd` to `retry_cmd_validate_expected`.